src/lm_saes/clt.py

```diff
@@ -1048,18 +1045,13 @@ def compute_loss(
 
         with timer.time("loss_calculation"):
             l_rec = (reconstructed - label).pow(2)
-            if use_batch_norm_mse:
-                l_rec = (
-                    l_rec
-                    / (label - label.mean(dim=1, keepdim=True)).pow_(2).sum(dim=-1, keepdim=True).clamp(min=1e-8).sqrt()
-                )
-            l_rec = l_rec.sum(dim=-1)
+            l_rec = l_rec.sum(dim=-1).mean()
```

Comment:
Crosscoder contains separate `l_rec` logging for individual heads. Early reduction of `l_rec` will run into error.